src/DependencyInjection/Compiler/TranslationPass.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/*
+ * This file is part of the DocusignBundle.
+ *
+ * (c) Grégoire Hébert <gregoire@les-tilleuls.coop>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace DocusignBundle\DependencyInjection\Compiler;
+
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Symfony\Component\DependencyInjection\Reference;
+use Symfony\Component\Translation\TranslatorInterface;
+
+final class TranslationPass implements CompilerPassInterface
```

Comment:
```suggestion
final class TranslationCompilerPass implements CompilerPassInterface
```

Reply:
then I should use the same for pluginpass